app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_user!, only: [:create]
+
+      def create
+        @book_suggestion = BookSuggestion.new(book_suggestion_params.merge(user_id: current_user.id))
```

Comment:
be aware, current_user.id will throw an undefined method for nil (NilClass) error, you should use the safe operator instead: `current_user&.id`

This means:

- If current user is nil, return nil
- If current user is **not** nil, return the value of current_user.id

Reply:
Done